src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
+
+public class LineDrawerAdapter implements Job2dDriver {
+    private int startX = 0, startY = 0;
+    private int lineType;
+
+    public LineDrawerAdapter(int lineType) {
+        this.lineType = lineType;
+    }
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine iLine;
+        if (lineType == 2) {
+            iLine = LineFactory.getDottedLine();
+        } else {
+            iLine = LineFactory.getSpecialLine();
```

Comment:
Hardcoded line types (no OCP).
Direct coupling with LineFactory class is unnecessary.